src/EditorFeatures/Test2/FindReferences/FindReferencesTests.DelegateInvokeMethodSymbols.vb

```diff
@@ -17,9 +17,9 @@ Imports System.Linq
  
 Module Program
     Function Main(args As String())
-        Dim q = Function(e As Integer)
+        Dim q = {|Definition:Function(e As Integer)
```

Comment:
THis test was simply bogus.  The test harness filtered out this span, but we do see this span when doing FAR or highlighting in the IDE.